src/sentry_utils.c

```diff
@@ -409,3 +422,54 @@ sentry__iso8601_to_msec(const char *iso)
 
     return (uint64_t)time * 1000 + msec;
 }
+
+#ifdef SENTRY_PLATFORM_WINDOWS
+#    define sentry__locale_t _locale_t
+#else
+#    define sentry__locale_t locale_t
+#endif
+
+static sentry__locale_t
+sentry__c_locale()
```

Comment:
For static, non exported functions I would not use the `sentry__` prefix. This implies the function exists outside of this module which it doesn't.